src/test/resources/fixtures/hmh/hmh-interacted/development/caliperEventAssessementStarted.json

```diff
@@ -0,0 +1,81 @@
+[
```

Comment:
this event isn't inside an array, so this should be removed.

Reply:
done!